tcp-native/src/main/java/org/ballerinalang/stdlib/tcp/SSLConfig.java

```diff
@@ -0,0 +1,363 @@
+/*
+ *  Copyright (c) 2021 WSO2 Inc. (http://wso2.com) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ *
+ */
+package org.ballerinalang.stdlib.tcp;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
```

Comment:
Above vertical space is needed ?

Reply:
removed

Reply:
seems this vertical space needed by checkstyle plugin, space not removed